lighthouse-core/lib/url-shim.js

```diff
@@ -70,16 +77,36 @@ URL.getDisplayName = function getDisplayName(url) {
     // Handle 'about:*' and 'data:*' URLs specially since they have no path.
     name = parsed.href;
   } else {
-    // Otherwise, remove any query strings from the path.
-    name = parsed.pathname.replace(/\?.*/, '')
-        // And grab the last two parts.
-        .split('/').slice(-2).join('/');
+    name = parsed.pathname;
+    const parts = name.split('/');
+    if (options.pathParts && parts.length > options.pathParts) {
+      name = '/\u2026/' + parts.slice(-1 * options.pathParts).join('/');
```

Comment:
Yea. Remove the first '/' would be good. Otherwise, it looks like a relative URL, up a directory.